src/Anexia.BDP.CloudLog/Client.cs

```diff
@@ -89,7 +82,7 @@ public Client(string index, string token)
         /// </param>
         public void SetClientType(string clientType)
```

Comment:
Remove method if possible.

Reply:
Not possible it's used by Serilog.Sinks.Cloudlog

Reply:
We will need to release a new version of `Serilog.Sinks.Cloudlog` anyway, so we remove the call to `SetClientType` there too.

Reply:
As discussed, we keep the `SetClientType` method as it is used by the `Serilog.Sinks.Cloudlog` library so set an all different client type.